webapp/src/components/NewsList/NewsList.vue

```diff
@@ -5,40 +5,41 @@
       <div v-for="item in orderedItems" :key="item.id">
         <NewsItem
           class="news-item"
-          @removeItem="removeItem(item)"
+          @updateItem="updateItem"
+          @removeItem="removeItem"
           :item="item"
         ></NewsItem>
       </div>
+      <div id="emptyListMessage" v-if="items.length === 0">
```

Comment:
More concise, but less strict about equality:
```suggestion
      <div id="emptyListMessage" v-if="!items.length">
```

Reply:
Good idea but we think it is less readable that way